src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/user/scheduling/CaptchaScheduler.java

```diff
@@ -0,0 +1,34 @@
+package com.sublinks.sublinksapi.api.lemmy.v3.user.scheduling;
+
+import com.sublinks.sublinksapi.api.lemmy.v3.user.repositories.CaptchaRepository;
+import com.sublinks.sublinksapi.api.lemmy.v3.user.services.CaptchaService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableScheduling
+@Configuration
+@RequiredArgsConstructor
+public class CaptchaScheduler {
+
+  private final CaptchaService captchaService;
+  private final CaptchaRepository captchaRepository;
+
+  @Scheduled(fixedRate=15*60*1000)
```

Comment:
JUST A NOTE:

It should be probably better if its changeable by the instance. As they should be able to tweak the restock time.

I honestly dont know how to do that but @jgrim could give a hint. Sorry! I am not that professional in java spring boot!